core/arch/arm/kernel/unwind_arm32.c

```diff
@@ -476,11 +476,14 @@ vaddr_t *unw_get_kernel_stack(void)
 	size_t exidx_sz = 0;
 	vaddr_t *tmp = NULL;
 	vaddr_t *addr = NULL;
-	struct unwind_state_arm32 state = { 0 };
```

Comment:
The preferred struct initialization is now `{ }`

Reply:
Will fix